app/controllers/users_controller.rb

```diff
@@ -2,6 +2,7 @@ class UsersController < ApplicationController
   def show
     @user = current_user
     @services = []
+    @user.services.each { |service| @services << service }
```

Comment:
Instead of pushing these to an array, you can do `@services = @user.services`, a bit more streamlined